### PR TITLE
fix: characters in MathML files

### PR DESCRIPTION
**Pull request by nschonni**
Might be intentional, but I noticed these pop up running spell checking on the new sections

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/math/acos/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ browser-compat: javascript.builtins.Math.acos
 
 The **`Math.acos()`** function returns the inverse cosine (in radians) of a number. That is,
 
-<math display="block"><semantics><mrow><mo>∀</mo><mi>x</mi><mo>∊</mo><mo stretchy="false">[</mo><mrow><mo>−</mo><mn>1</mn></mrow><mo>,</mo><mn>1</mn><mo stretchy="false">]</mo><mo>,</mo><mspace width="0.2777777777777778em"></mspace><mrow><mo lspace="0em" rspace="0.16666666666666666em">𝙼𝚊𝚝𝚑.𝚊𝚌𝚘𝚜</mo><mo stretchy="false">(</mo><mi>𝚡</mi><mo stretchy="false">)</mo></mrow><mo>=</mo><mo lspace="0em" rspace="0em">arccos</mo><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>=</mo><mtext>the unique&nbsp;</mtext><mi>y</mi><mo>∊</mo><mo stretchy="false">[</mo><mn>0</mn><mo>,</mo><mi>π</mi><mo stretchy="false">]</mo><mtext>&nbsp;such that&nbsp;</mtext><mo lspace="0em" rspace="0em">cos</mo><mo stretchy="false">(</mo><mi>y</mi><mo stretchy="false">)</mo><mo>=</mo><mi>x</mi></mrow><annotation encoding="TeX">\forall x \in [{-1}, 1],\;\mathtt{\operatorname{Math.acos}(x)} = \arccos(x) = \text{the unique } y \in [0, \pi] \text{ such that } \cos(y) = x</annotation></semantics></math>
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/math/acos/index.md:16]: This is intended. It's output by http://fred-wang.github.io/TeXZilla/ when you input `\mathtt`. I don't see something clearly wrong with it.

Comment (nschonni) [reply files/en-us/web/javascript/reference/global_objects/math/acos/index.md:16]: Sorry, I think GitHub isn't showing the comment correctly, but this is replacing `𝙼𝚊𝚝𝚑.𝚊𝚌𝚘𝚜` with the ASCII version `Math.acos`, not touching any of the `\mathtt` parts

Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/math/acos/index.md:16]: Yep, I'm talking about the markup. The characters aren't showing up as monospace anymore.

Comment (nschonni) [reply files/en-us/web/javascript/reference/global_objects/math/acos/index.md:16]: Unfortunate, I do see the rending difference in the preview, so I'll close this

**Pull request closed without merging**